### ISSUE TITLE
[Feature Request]: Krypton version of `WebView2`

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#1026]
Is it possible to add Krypton to `WebView2` in a similar way to the `KryptonWebBrowser`? https://learn.microsoft.com/en-gb/microsoft-edge/webview2/

**Comment by Smurf-IV** [comment]
Not sure "How much" wrapping can be done with that control.
i.e. put a border round it, then what ?
A border can be simply done already via a panel.

**Comment by Wagnerp** [comment]
The documentation isn't that complete. As I understand it, WebView2 is based on the newer Edge (chromium) tech, though I may be wrong...

**Comment by Smurf-IV** [comment]
So, What is expected to happen with the Kryptonisation then ?

**Comment by Wagnerp** [comment]
> So, What is expected to happen with the Kryptonisation then ?

Context Menus etc. I think the existing web browser winforms control is still based on Internet Explorer. No idea why they haven't updated it to use the current tech?

**Comment by Smurf-IV** [comment]
> the existing web browser winforms control is still based on Internet Explorer.

That is correct, so that "Old" programs still compile and work with the DCom way of working.

WebView is completely different.
And thus requires a different Way of Working.
As for menus etc. then these events (I think) can still be captured as normal and use existing KComponents via a standard redirect.

**Comment by PWagner1** [comment]
@Smurf-IV, @giduac & @tobitege 

Ok, so some extra build steps will be needed to fetch the appropriate DLLs (don't worry, this will be automated via `run.cmd`) :).
